suite-native/module-connect-device/src/components/PinOnDevice.tsx

```diff
@@ -0,0 +1,79 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { Dimensions } from 'react-native';
+import { useCallback, useEffect } from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Image, Box, Text } from '@suite-native/atoms';
+import { DeviceModelInternal } from '@trezor/connect';
+import {
+    removeButtonRequests,
+    selectDevice,
+    selectDeviceModel,
+    selectDeviceRequestedPin,
+} from '@suite-common/wallet-core';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import TrezorConnect, { UI_REQUEST } from '@trezor/connect';
+
+const SCREEN_HEIGHT = Dimensions.get('screen').height;
+
+export const deviceImageMap = {
+    [DeviceModelInternal.T1B1]: require('../assets/pin-t1b1.png'),
+    [DeviceModelInternal.T2T1]: require('../assets/pin-t2t1.png'),
+    [DeviceModelInternal.T2B1]: require('../assets/pin-t2b1.png'),
+} as const satisfies Record<DeviceModelInternal, string>;
+
+const wrapperStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingTop: utils.spacings.large,
+}));
+const imageStyle = prepareNativeStyle(_ => ({
+    maxHeight: SCREEN_HEIGHT * 0.6,
+    width: 243,
+    height: 550,
+    alignItems: 'center',
+    contentFit: 'contain',
+}));
+
+export const PinOnDevice = () => {
+    const dispatch = useDispatch();
+    const navigation = useNavigation();
+
+    const device = useSelector(selectDevice) ?? null;
```

Comment:
`?? null` redundant?

Reply:
yes, it was 42c4f8d5afc50de12b9184edd5ead07bc01862be & 387057fc42f64115d98055b3a42eb51c4d99c60f